orocos_kdl/src/joint.hpp

```diff
@@ -72,10 +74,12 @@ namespace KDL {
          * @param inertia 1D inertia along the joint axis, default: 0
          * @param damping 1D damping along the joint axis, default: 0
          * @param stiffness 1D stiffness along the joint axis,
+         * @param upper_position_limit upper position limit,
+         * @param lower_position_limit lower position limit,
          * default: 0
          */
-        explicit Joint(const JointType& type=Fixed,const double& scale=1,const double& offset=0,
-               const double& inertia=0,const double& damping=0,const double& stiffness=0);
+        explicit Joint(const JointType& type=None,const double& scale=1,const double& offset=0,
```

Comment:
```suggestion
        explicit Joint(const JointType& type=Fixed, const double& scale=1, const double& offset=0,
```

Wrong merge conflict resolution? `Fixed` is preferred over `None` since https://github.com/orocos/orocos_kinematics_dynamics/commit/dce6742a379698bc6d50afedc8c0a4fc4657d493.